hack/generator/pkg/astmodel/package_definition.go

```diff
@@ -9,10 +9,11 @@ import (
 	"bytes"
 	"fmt"
 	"io/ioutil"
-	kerrors "k8s.io/apimachinery/pkg/util/errors"
 	"path/filepath"
 	"text/template"
 
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+
```

Comment:
You shouldn't have a break between `kerrors` and `github.com/pkg/errors`. Generally the pattern is to have (up to) 3 import stanzas: stdlib, external packages, this-module packages.